src/locations/model/Location.ts

```diff
@@ -4,4 +4,5 @@ export default interface Location {
   businessId: string;
   createdAt: Date;
   updatedAt: Date;
+  aggregateLocationIds: string[];

```

Comment:
Thanks for adding this @clericuzzi . Since you are adding here, can you also add to location create/update requests, since these are writeable fields there. This will help bates if it's already there when he implements the webapp story around this.

https://github.com/ChannelApe/channelape-typescript-sdk/blob/master/src/locations/model/LocationUpdateRequest.ts

https://github.com/ChannelApe/channelape-typescript-sdk/blob/master/src/locations/model/LocationCreateRequest.ts